addOns/imagelocationscanner/src/main/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRule.java

```diff
@@ -68,6 +68,10 @@ public int getPluginId() {
 		return PLUGIN_ID;
 	}
 
+    public String getHelpLink() {
+        return "https://www.zaproxy.org/docs/desktop/addons/imagelocationscanner/#id-" + PLUGIN_ID;
```

Comment:
`https://www.zaproxy.org/docs/desktop/addons/image-location-and-privacy-scanner/`